app/src/main/java/com/celzero/bravedns/ui/UniversalFirewallFragment.kt

```diff
@@ -157,7 +157,8 @@ class UniversalFirewallFragment : Fragment(R.layout.universal_fragement_containe
         }
 
         includeView.firewallAppsShowTxt.setOnClickListener {
-            openCustomIpDialog()
+            enableAfterDelay(TimeUnit.SECONDS.toMillis(2L), includeView.firewallAppsShowTxt)
```

Comment:
isn't there a constant declared for this `SHORT_DELAY` or some such??

Reply:
No, we don't have that. Will introduce something like that. There are other places where we are using this. Will replace those as well after adding the constants.